packages/core/test/unit/html/NodeProcessor.test.ts

```diff
@@ -122,6 +123,74 @@ test('processNode processes dropdown with header slot taking priority over heade
                            testData.PROCESS_DROPDOWN_HEADER_SLOT_TAKES_PRIORITY_EXPECTED);
 });
 
+test('processNode does not log warning when lazy pic has width or height',
+     () => {
+       const nodeProcessor = getNewDefaultNodeProcessor();
+
+       const testCode = '<pic scr="" alt="" width="300" lazy></pic>';
+       const testNode = parseHTML(testCode)[0] as MbNode;
+
+       const consoleSpy = jest.spyOn(logger, 'warn');
+
+       nodeProcessor.processNode(testNode, new Context(path.resolve(''), [], {}, {}));
+
+       const logMessage = consoleSpy.mock.calls[0];
```

Comment:
I think you can write an expect statement like `expect(consoleSpy.warn).toHaveBeenCalledWith(undefined);` and combine these two

Reply:
Thanks for the suggestion!
I am thinking of another way to do it `expect(consoleSpy).not.toHaveBeenCalled()`. 
It checks if the `warn` method is not being called at all